src/main.rs

```diff
@@ -23,15 +24,42 @@ const PLATFORM: &'static [PlatformData] = &[
 fn main() {
     let mut col = Collector::new();
 
+	let matches = App::new("conga")
+        .version("0.1.0")
+        .author("")
```

Comment:
same here

Reply:
version can be pulled using the `crate_version!` macro provided by clap. ignoring author, it doesn't matter